external/java/src/test/java/ch/epfl/dedis/byzcoin/ByzCoinRPCTest.java

```diff
@@ -56,6 +56,32 @@ void ping() {
         assertTrue(bc.checkLiveness());
     }
 
+    @Test
+    void getBlocks() throws Exception {
```

Comment:
I don't see how this code will exercise the part of the code about doing multiple get update chains, and linking from the last known update to the first block of the second get update chain call.

Reply:
you're right, this doesn't exercise the part the multiple get update chains. trying to write a test that will do it, probably something with roster change

Reply:
testing the getUpdateChain loop turned out to be quite tricky. I had to implement some new skipchain APIs to create skipblocks the same way as the test in go. the only way to make skipblocks before was using the byzcoin API